homeassistant/components/zha/light.py

```diff
@@ -1049,6 +1049,20 @@ def __init__(
         """Initialize a light group."""
         super().__init__(entity_ids, unique_id, group_id, zha_device, **kwargs)
         group = self.zha_device.gateway.get_group(self._group_id)
+
+        self._GROUP_SUPPORTS_EXECUTE_IF_OFF = True  # pylint: disable=invalid-name
```

Comment:
Should this maybe just be called `_group_supports_execute_if_off` (instead of upper-casing)?

(I originally upper-cased it, as it was somewhat similar to the config values that are set when initializing the light classes and it's also done there (likely because it's somewhat of a constant?))